apps/server/src/modules/task/controller/task.controller.ts

```diff
@@ -7,7 +7,7 @@ import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator
 // todo  @src/modules/learnroom/* must be replaced
 import { CopyApiResponse } from '@src/modules/learnroom/controller/dto/copy.response';
 import { CopyMapper } from '@src/modules/learnroom/mapper/copy.mapper';
-import serverConfig from '@src/modules/server/server.config';
+import { serverConfig } from '@src/modules/server/server.config';
```

Comment:
I like to put it also to a index file for '@src/modules/server' to decouple the internal sorting.

Reply:
I had problem with circular dependencies